### DISCUSSION TITLE
Disabling Tamagui Compiler in development

### DISCUSSION THREAD

**Discussion by matthewhausman** [tamagui/tamagui#766] [Q&A]
@natew 

In the recent pod you did you recommended turning off the tamagui compiler during development and also discusses a number of ways to do so

Do you mind elaborating a bit here or pointing to the section of the docs that discusses those methods? Tysm

**Comment by natew** [comment]
For example with next.js:

https://github.com/tamagui/tamagui/blob/master/apps/site/next.config.js#L33

Otherwise can just not add the compiler plugin based on NODE_ENV

**Reply by matthewhausman** [reply]
sick - how about the per file - basis command? "pragma" something or rather?

**Reply by natew** [reply]
// tamagui-ignore

That pragma will disable it per-file.
